javascript-objects/main.js

```diff
@@ -0,0 +1,36 @@
+var student = {
+  firstName: 'Denny',
+  lastName: 'Min',
+  age: 23
+};
+
+student.livesInIrvine = true;
+student.previousOccupation = 'customer service rep';
+console.log('value of livesInIrvine: ' + student.livesInIrvine);
+console.log('value of previousOccupation: ' + student.previousOccupation);
+
+var fullName = student.firstName + ' ' + student.lastName;
+console.log('value of fullName: ' + fullName);
+console.log('value of student: ', student);
+
+var vehicle = {
+  make: 'honda',
+  model: 'civic',
+  year: '2020'
+};
+
+vehicle.color = 'dark gray';
+vehicle.isConvertible = false;
```

Comment:
Please use square bracket notation, as requested in the directions
![image](https://user-images.githubusercontent.com/32880187/123709632-3b481700-d822-11eb-9c79-f85d1d49d317.png)